dm/dm/config/task.go

```diff
@@ -377,9 +406,10 @@ type TaskConfig struct {
 	BWList map[string]*filter.Rules `yaml:"black-white-list" toml:"black-white-list" json:"black-white-list"`
 	BAList map[string]*filter.Rules `yaml:"block-allow-list" toml:"block-allow-list" json:"block-allow-list"`
 
-	Mydumpers map[string]*MydumperConfig `yaml:"mydumpers" toml:"mydumpers" json:"mydumpers"`
-	Loaders   map[string]*LoaderConfig   `yaml:"loaders" toml:"loaders" json:"loaders"`
-	Syncers   map[string]*SyncerConfig   `yaml:"syncers" toml:"syncers" json:"syncers"`
+	Mydumpers  map[string]*MydumperConfig  `yaml:"mydumpers" toml:"mydumpers" json:"mydumpers"`
+	Loaders    map[string]*LoaderConfig    `yaml:"loaders" toml:"loaders" json:"loaders"`
+	Syncers    map[string]*SyncerConfig    `yaml:"syncers" toml:"syncers" json:"syncers"`
+	Validators map[string]*ValidatorConfig `yaml:"validators" toml:"validators" json:"validators"`
```

Comment:
why we need different `Validators` for different source ? will user want it ? e.g: one souce use `full` but another use `fast` ？

Reply:
btw i perfer to not put this config into subtask config, this two things can be unnecessarily coupled to each other 

seems user can just use ( dmctl validator start/stop --task ---source --mode) to operate validator ?

Reply:
According to the current design, the user can start a validator when starting the task and thus the validator config is coupled with the task's. 
Referring to the spec, they can specify different validators for different sources. It might possibly be more accessible when they prefer a `full` validator working on small database while a `fast` validator working on a big database.

Reply:
> why we need different Validators for different source ? will user want it ?

may not, then just use the same validation config. right now there's only one param in `ValidatorConfig`, may add more later, user may want different parameter for different source, just like other units

> btw i perfer to not put this config into subtask config

validation is a `subtask` level unit just like `syncer`, it can only run when there's a `syncer` unit in this subtask actually.

Reply:
i found a user sccenario: user want start validation as soon as the sync phase is entered, so this lgtm